src/command.py

```diff
@@ -49,6 +49,43 @@ def __repr__(self):
                f"Gem message: {self.gem_message}"
 
 
+class GemsCounterFromMessage:
```

Comment:
Do we really need a whole class for this?

Reply:
i had the same question. decided to create a class considering 2 things:
- making a class gives more manageability in terms of adding more templates to give gems. we can do that if we want.
- why not, making a class does harm in anyway.
please let me know if you have other thoughts in mind. i only shared my thoughts while coding, may be missing something?

Reply:
Semantically speaking a class makes sense when you have locallised attributes that belong to the instance of that class, in this case we are not encapsulating/locallising anything, infact, what would be better if we create a separate module/file in python called gem-message-template-parser.py which can have an entry function that gets passed to the gemMessage class, meaning in the future, we can switch that out for a different gem message parser. Move those tiny functions into a file, create a dependency injection system for the gemmessage whatever class and it might look really clean :D

Reply:
based on discussion, moved the message parsing implementation to a separate file in commit [ef68eb3](https://github.com/CraftsmenLtd/DiscordGems/pull/30/commits/ef68eb35c6dbc07eab17f1ab69a754f1788c3240)